test/e2e/wizard.test.js

```diff
@@ -0,0 +1,169 @@
+const { assert, driver, By } = require('vl-ui-core').Test.Setup;
+const { VlH2, VlH3 } = require('vl-ui-titles').Test.VlTitles;
+const { VlElement } = require('vl-ui-core').Test;
+const { VlCheckbox } = require('vl-ui-checkbox').Test;
+const VlWizardPage = require('./pages/vl-wizard.page');
+
+describe('vl-wizard', async () => {
+    const vlWizardPage = new VlWizardPage(driver);
+
+    before(() => {
+        return vlWizardPage.load();
+    });
+
+    afterEach(async () => {
+        await vlWizardPage.reset();
+    });
+
+    it('als gebruiker kan ik de titel en header van de wizard zien', async () => {
+        const wizard = await vlWizardPage.getWizard();
+        const titleSlotElements = await wizard.getTitleSlotElements();
+        const headerSlotElements = await wizard.getHeaderSlotElements();
+        const title = await new VlH2(driver, titleSlotElements[0]);
+        const header = await new VlElement(driver, headerSlotElements[0]);
+        await assert.eventually.isTrue(title.isH(2));
+        await assert.eventually.equal(title.getText(), 'Wizard title');
+        await assert.eventually.equal(header.getText(), "You're a wizard Harry");
+    });
+
+    it('als gebruiker kan ik alleen het actief paneel van de wizard zien', async () => {
+        const wizard = await vlWizardPage.getWizard();
+        const pane1 = await wizard.getPane(1);
+        const pane2 = await wizard.getPane(2);
+        const pane3 = await wizard.getPane(3);
+        const pane4 = await wizard.getPane(4);
+        await assert.eventually.isTrue(pane1.isActive());
+        await assert.eventually.isFalse(pane2.isActive());
+        await assert.eventually.isFalse(pane3.isActive());
+        await assert.eventually.isFalse(pane4.isActive());
+    });
+
+    it('als gebruiker kan ik via de progress bar doorheen de wizard panelen navigeren', async () => {
```

Comment:
Mierenneuken, maar zulke grote blokken code lezen voor mij moeilijker zonder witregels tussen logische groeperingen van statements. Het groeperen per stap en er een witregel tussenplaatsen verhelpt dat meestal. Zelfde opmerking voor enkele andere testen in deze file.

Reply:
Ik heb wat enters toegevoegd, hopelijk is het nu leesbaarder.

Reply:
Dank u, koning van de newline